src/hooks/transactions/useParseSignedTransactions.tsx

```diff
@@ -43,6 +44,7 @@ export function useParseSignedTransactions(
         }
 
         if (signedTransactions.length > 0) {
+          // TODO: check if the transactions are same as the ones in the redux store
```

Comment:
This TODO will be resolved in the same version or in the next version?

Reply:
Will be resolved in a next PR yes